fitlins/interfaces/bids.py

```diff
@@ -232,9 +232,8 @@ def _load_level1(self, runtime, analysis):
                 # t/aCompCor
                 # We may want to add criteria to include HPF columns that are not
                 # explicitly listed in the model
-                names = [col for col in dense.columns
-                         if col.startswith('non_steady_state') or
-                         col in step.model['x']]
+                names = step.model['x'].copy()
```

Comment:
What logical groupings? You mean order that you put into the model yourself? I'm fine with trying to preserve the order in any case, just confusing what you're referring to.

Reply:
Yes, just preserving the model writer's specification. I suspect most people will group related variables together and variables of interest first, so that's all I meant by logical groupings.